### PR TITLE
W-11076381 Added new permission information for Functional Monitoring 

### PR DESCRIPTION
**Pull request by bjoson-mulesoft**

### CODE REVIEW

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 1]
```diff
@@ -9,7 +9,16 @@ This procedure describes creating monitors via the Functional Monitoring UI.
 
 == Before You Begin
 
-* Ensure that your user ID has the Anypoint Monitoring User permission and that it is assigned the Exchange Administrators role. An administrator for your organization can grant this permission in Anypoint Access Management. For details, see xref:access-management::roles.adoc[Roles].
+* Ensure that you have the following permissions set in Access Management:
++
+** *Anypoint Monitoring > Viewer* or *Anypoint Monitoring > Administrator*
```
Comment (florvimberg) [inline modules/ROOT/pages/afm-create-monitor.adoc:14]: Hi Beth! Where did you get that this permission is needed? We didn't have it in our radar

**Review by florvimberg — COMMENTED** [review]
Hi Beth, @Christian-Vallejos and I reviewed the changes and only have a question regarding the Anypoint Monitoring > Viewer permission. The rest looks great

File: modules/ROOT/pages/afm-create-monitor.adoc (hunk 1 above)
Comment (bjoson-mulesoft) [reply modules/ROOT/pages/afm-create-monitor.adoc:14]: Hi @florvimberg. The current doc has in the first bullet [here](https://docs.mulesoft.com/api-functional-monitoring/afm-create-monitor#before-you-begin) that "Anypoint Monitoring User" is required. There is no such permission that I could see, so I listed the two permissions that there actually are. I also added this to the update monitor topic, as I mentioned in this [Slack thread](https://docs.mulesoft.com/api-functional-monitoring/afm-create-monitor#before-you-begin).
Should I remove both of the Anypoint Monitoring permissions in both places? Or are you saying they need the Anypoint Monitoring > Administrators permission? Please let me know. Thank you.

Comment (bjoson-mulesoft) [reply modules/ROOT/pages/afm-create-monitor.adoc:14]: Thanks for your reviews. I've posted a screenshot of the permission in question in the Slack thread.

Comment (florvimberg) [reply modules/ROOT/pages/afm-create-monitor.adoc:14]: Hi Beth! Thanks for the info, Chris and I reviewed it and think it is ok to add the permissions as you said! Thanks for the work, let us know when you will be merging this PR

**Review by sathya0 — DISMISSED** [review]

File: modules/ROOT/pages/afm-create-monitor.adoc (hunk 1 above)
Comment (bjoson-mulesoft) [reply modules/ROOT/pages/afm-create-monitor.adoc:14]: Thanks, Maria and Chris. I've posted in Slack that I can merge this early Monday am to coordinate with the service deployment.

**Review by sathya0 — APPROVED** [review]

**Pull request merged**